scripts/run_in_file_program.sh

```diff
@@ -1,5 +1,12 @@
-set -e
+set -eu
 BASE_PATH=$(pwd)
-cd workspace/$CHALLENGE_NAME
-docker run --rm -v $BASE_PATH/challenges/$CHALLENGE_NAME/input.txt:/data/input.txt $CHALLENGE_NAME /data/input.txt  > result.txt
 
+cd workspace/$CHALLENGE_NAME
+rm -rf data/
```

Comment:
After every run, the workspace will cleanup, so this looks like to be redundant.

https://github.com/soallpeach/onboarding/blob/17338f1a18051c9101cf021f1fef3d6dbb213166/scripts/cleanup.sh#L3